imblearn/pipeline.py

```diff
@@ -149,8 +149,7 @@ def _validate_steps(self):
 
             if isinstance(t, pipeline.Pipeline):
                 raise TypeError(
-                    "All intermediate steps of the chain should not be"
-                    " Pipelines"
+                    "All intermediate steps of the chain should not be" " Pipelines"
```

Comment:
This should be switched to: `"All intermediate steps of the chain should not be Pipelines"`